docs/database_schema_diagrams/write_graphs.pl

```diff
@@ -203,7 +204,7 @@ sub write_dot_files
         unless (contains_string(\@created_tables, $table)) {
             log_warning { "The table '$table' in '$diagram_id' diagram doesn't exist." };
         }
-        if (contains_string([ %$diagram_tables_props{$table} ], 'shortened')) {
+        if (contains_string(\@{ %$diagram_tables_props{$table} }, 'shortened')) {
```

Comment:
It doesn't seem intended to take a hash slice here, unless I'm missing something. So this should just be:

```Perl
        if (contains_string($diagram_tables_props->{$table}, 'shortened')) {
```

Reply:
Good catch! I’ve added the commit https://github.com/metabrainz/musicbrainz-server/pull/3308/commits/354f497a38fcf5417eac2d5e2b840e65fdc4ab44 to the pull request #3308.